sdk/servicefabric/Azure.ResourceManager.ServiceFabric/api/Azure.ResourceManager.ServiceFabric.netstandard2.0.cs

```diff
@@ -355,14 +362,14 @@ public partial class ApplicationUpgradePolicy
         public bool? ForceRestart { get { throw null; } set { } }
         public bool? RecreateApplication { get { throw null; } set { } }
         public Azure.ResourceManager.ServiceFabric.Models.ArmRollingUpgradeMonitoringPolicy RollingUpgradeMonitoringPolicy { get { throw null; } set { } }
-        public Azure.ResourceManager.ServiceFabric.Models.RollingUpgradeMode? UpgradeMode { get { throw null; } set { } }
-        public string UpgradeReplicaSetCheckTimeout { get { throw null; } set { } }
+        public Azure.ResourceManager.ServiceFabric.Models.ApplicationRollingUpgradeMode? UpgradeMode { get { throw null; } set { } }
+        public System.TimeSpan? UpgradeReplicaSetCheckTimeout { get { throw null; } set { } }
     }
     public partial class ApplicationUserAssignedIdentity
```

Comment:
Where does this class get used? This looks like a thing that should be replaced

Reply:
It has `Name` property instead of `ClientId`. When it's used there is no corresponding ResourceId for the identity. I guess it's probably designed differently than the common type UserAssignedIdentity.

Reply:
This is much simple than the `ManagedServiceIdentity` in RM, so would wait Feng's change